lib/config.js

```diff
@@ -100,7 +100,7 @@ require('./credentials/credential_provider_chain');
  *     Currently supported options are:
  *
  *     * **base** [Integer] &mdash; The base number of milliseconds to use in the
- *       exponential backoff for operation retries. Defaults to 100 ms.
+ *       exponential backoff for operation retries. Defaults to 30 ms.
```

Comment:
The default used to be 30 ms but was changed to 100 ms with the configurable retry delays.